llvm-remote-index.md

```diff
@@ -0,0 +1,79 @@
+# LLVM remote index service
+
+We plan to provide [remote index](/remote-index.md) service for LLVM
+developers, this document describes the details of this service.
+
+We provide our best effort support for remote index service but it is not an
+official product by either LLVM or Google.
+
+**NOTE**: The service is currently in the alpha testing stage and we would
+like to gradually expand the scope of availability and feature set.
+
+## Overview
+
+LLVM remote index service is implemented by the team maintaining clangd. We
+plan to provide the service to LLVM developers to lower the entry bar for
+developers with less powerful machines and make the development process more
+accessible. The whole service implementation, including infrastructure, is
+open source and publicly available
+([clangd/index/remote](https://github.com/llvm/llvm-project/tree/master/clang-tools-extra/clangd/index/remote)
+for `clangd-index-server` and client implementation,
+[llvm-remote-index](https://github.com/clangd/llvm-remote-index) for
+indexing, data pipelines and deployment). Google donated Google Cloud
+Platform instance for us to run the service.
+
+We index the whole LLVM codebase (`-DLLVM_ENABLE_PROJECTS="all"`) on a daily
+basis, index snapshots are available in [clangd/llvm-remote-index
+releases](https://github.com/clangd/llvm-remote-index/releases). We only
+index the main branch of LLVM and check for the new idex version every 6
+hours on the deployment server. The server also checks for new versions of
+`clangd-index-server` which is fetched from [clangd/clangd
+snapshots](https://github.com/clangd/clangd/releases). If you use LLVM remote
```

Comment:
i would drop the suggestion around using snapshots.

Reply:
and the rest about breaking changes, they are not about LLVM remote-index instance. they are about the remote-index design itself.